packages/web3auth/src/index.ts

```diff
@@ -48,7 +48,7 @@ function web3auth(options: Web3AuthModuleOptions): WalletInit {
           ticker: token,
           tickerName: label,
           chainId: id,
-          rpcTarget: rpcUrl,
+          rpcTarget: rpcUrl ?? "", // TODO: Ask maintainers if we should throw instead.
```

Comment:
Is the rpcUrl prop required or are there defaults per chain for the web3Auth provider?

Reply:
I don't believe web3auth has defaults per chain.

Reply:
and `rpcTarget` is required.

Reply:
(at least according to the type system)

Reply:
OK their documentation is wrong, they do require an rpcTarget. I suggest passing the value to web3auth and let it fail on their end. It's going to throw no matter what.

![image](https://github.com/blocknative/web3-onboard/assets/64225318/c5bb6e90-4375-4b62-9959-14c58b4db3fd)